### ISSUE TITLE
Ability to quickly copy & paste styles, not just settings

### ISSUE THREAD

**Issue by hampshirv** [beaverbuilder/feature-requests#379]
The ability to quickly copy and paste row, column, modules, etc settings (example below) has been a great addition, but a lot of the time I would like to copy and paste their styles, NOT all of the settings (which is often content). If anything, this would be more useful to me than the overall 'Copy Settings'.

![image](https://github.com/user-attachments/assets/13847c61-86b1-46b7-b553-553a8f791878)

Although I've not tested it extensively, I think it's already possible via a more long-winded method (Advanced > Export/Import > Copy Styles). **This request is to add it to the item controls, so it's much quicker to do.**

Looking back at old requests, I think this is related to issues [210](https://github.com/beaverbuilder/feature-requests/issues/210) and [75](https://github.com/beaverbuilder/feature-requests/issues/75).

**Comment by codente** [comment]
The issue with this is we currently do not have a way to define a setting as a "style" so there's no easy way to grab only the styles. I mentioned this in #75.  We are looking at a feature (Module Design Tab) where we could do this for modules that support that Design tab but we're not there yet.

**Comment by hampshirv** [comment]
> The issue with this is we currently do not have a way to define a setting as a "style" so there's no easy way to grab only the styles. I mentioned this in #75. We are looking at a feature (Module Design Tab) where we could do this for modules that support that Design tab but we're not there yet.

Yeah, I assumed that might still be a hindrance from before, although did wonder how close this already was (hands up, I don't use it much because it's a bit buried)
![image](https://github.com/user-attachments/assets/aa62392f-e8cd-4bdd-a72d-07f51095d0b8)

**Comment by pixelloop** [comment]
I agree, being able to copy the styles more quickly from one module, box or row to another, without the full settings would be a great quality of life improvement. 

While I much prefer Beaver Builder, Elements use of right click anywhere on the module to copy and paste styles is the perfect implementation for this.
